src/App.tsx

```diff
@@ -1,61 +1,155 @@
+import { FormEvent, useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(todoUserId: number): User | null {
```

Comment:
don't forget about your pasts checlists
```suggestion
function getUserById(id: number): User | null {
```